File: BarNone/TheBar/src/App/Data/Models/Analysis/AnalysisResult.ts
import { ELiftAnalysisType } from "App/Data/Models/Analysis/AnalysisRequest";

export default class AnalysisResult {
	public Results: ResultEntity[];
}

export class ResultEntity {
	[key: string]: any;

	public type: ELiftAnalysisType;

	public value: any;
}

File: BarNone/Shared/BarNone.Shared.Analysis/LiftAnalysisPipeline/Core/AnalysisResult.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;
using System.Collections.Generic;
using System.Text;

namespace BarNone.Shared.Analysis.LiftAnalysisPipeline.Core
{
    public class AnalysisResult
    {
        public AnalysisResult()
        {
            Results = new List<ResultEntity>();
        }

        public List<ResultEntity> Results { get; set; }
    }

    public class ResultEntity
    {
        public ELiftAnalysisType Type { get; set; }

        public object Value { get; set; }
    }
}
